webpack/base.config.js

```diff
@@ -59,6 +59,7 @@ module.exports = {
             {test: /\.(jpe?g|png)/, use: 'url-loader?limit=4096'},
             {test: /\.json$/, use: 'json-loader'},
             {test: /\.js$|\.jsx$/, exclude: /node_modules/, use: 'babel-loader'},
+            // All svgs are loaded inline...
```

Comment:
put this in another pr where we address the removal of svgs from the webpack stats output